ui/src/EventForm.tsx

```diff
@@ -1,43 +1,53 @@
 import React, { useState } from "react";
-import { formatDate, getDateTimeString, padNumberWith0Zero } from "./lib";
+import {
+  formatDate,
+  getDateTimeString,
+  padNumberWith0Zero,
+  timeFormat,
+} from "./lib";
 import { Checkbox, RadioGroup, Radio, Stack } from "@chakra-ui/react";
 import s from "./EventForm.module.css";
 
 interface FormProps {
-  initialStartDate: string;
-  initialEndDate: string;
-  initialStartTime: string;
-  initialEndTime: string;
+  initialStart: string; // ex. "2023-01-25T08:00:00.000Z"
+  initialEnd: string; // ex. "2023-01-25T09:00:00.000Z"
   initialTitle: string;
   initialDescription: string;
   initialAllDay: boolean;
   initialRecurring: boolean;
-  initialRecurrenceEnd: string;
+  initialRecurrenceEnd: string; // ex. "2023-01-25T08:00:00.000Z"
```

Comment:
are these comments serving as examples? If so we could create tests that pass this format as documentation (if we don't have them already).

Reply:
+1